src/languages/es.json

```diff
@@ -0,0 +1,118 @@
+{
+    "common": {
+        "copy": "Copiar",
+        "open": "Abrir",
+        "send": "Enviar",
+        "cancel": "Cancelar",
+        "account": "Cuenta",
+        "amount": "Cantidad",
+        "total_amount": "Cantidad Total",
+        "your_account": "Mi Cuenta",
+        "public_address": "Dirección Pública",
+        "secret_seed": "Semilla Secreta",
+        "receive": "Recibir",
+        "account_address": "Dirección de la cuenta",
+        "generator": "Generador",
+        "close": "Cerrar",
+        "transaction_fee": "Comisión de la transacción",
```

Comment:
What about modifying this with [comisiones de transacción]?

Transaction fee translation reference 
  https://blog.bit2me.com/es/explorador-de-blockchain-transacciones/